<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonRenovar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>671</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Renovar Suscripción</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../recursos.qrc">:/imagenes/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCancelar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>671</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar Suscripción</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHistorial">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>671</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver historial de entrenamientos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRutina">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>671</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Solicitar rutina de entrenamiento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDieta">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>671</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Solicitar dieta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClase">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>671</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Apuntarse a clase grupal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCerrar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerrar sesión</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../recursos.qrc"/>
  <include location="../../../../../../../../../../recursos.qrc"/>
 </resources>
 <connections/>
</ui>
